linera-core/src/unit_tests/wasm_client_tests.rs

```diff
@@ -757,20 +760,12 @@ where
     let request_subscribe = social::Operation::Subscribe {
         chain_id: sender.chain_id(),
     };
-    let cert = receiver
+    receiver
         .execute_operation(Operation::user(application_id, &request_subscribe)?)
         .await
         .unwrap()
         .unwrap();
```

Comment:
The double unwrap is a little unclear here since we are no longer using the returned value.
Therefore, maybe it is better to write
```rust
assert!(matches!(outcome, ClientOutcome::Committed(_)));
```

Maybe several other instances here.

Reply:
How do you feel about an `unwrap_ok_committed` method for an extension trait of `Result<ClientOutcome<_>>`? That would turn the double `unwrap`s into a single, somewhat more readable line everywhere.

Reply:
Done, but I'll make a separate PR for this, too because it's going to be more changed lines than this one.

Reply:
As you prefer.
I do not like the `fn unwrap()` given that it creates some ambiguity.